static/styles/compose.css

```diff
@@ -228,26 +228,32 @@
 
     button {
         background: transparent;
-        font-size: 20px;
-        font-weight: bold;
-        line-height: 20px;
         opacity: 0.6;
-        border: 0;
-        padding: 0;
         margin-left: 4px;
-        vertical-align: unset;
+        border: 0;
 
         &:hover {
             opacity: 1;
         }
     }
 }
 
-.collapse_composebox_button,
-#compose_close {
+.collapse_composebox_button {
     display: none;
 }
 
+.composebox-arrow {
+    font-size: 20px;
+    margin-top: -1px;
+    font-weight: bold;
+}
+
+#compose_close {
+    font-weight: lighter !important;
+    text-shadow: none;
+    font-size: 15px;
+}
```

Comment:
I think it'd be better to scope these inside the `compose_top_right` block, for readability.